Update project status on readme (#178)
HelloYeew/maisim: README.md

## Project Status

> The UI and design of the game are starting get in pieces now. The gameplay style are still not much finalized, but it's getting there.

Reviewer: ```suggestion
The UI and design pieces  of the game are slowly getting done. Gameplay design isn't finalized yet, but we're getting there.
```
